catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -99,6 +99,13 @@
             default="false"
             required="false"/>
 
+        <AD id="cacheDisabled"
+            name="Disable Cache"
+            description="Disables use of cache."
```

Comment:
I think this would be more helpful if it was something along the lines of "Disables the presence of cached results when running queries."